src/web/clients/requests/get.md

```diff
@@ -8,32 +8,25 @@ status and headers. Reads HTTP response body into an allocated [`String`]
 using [`read_to_string`].
 
 ```rust,no_run
-# #[macro_use]
-# extern crate error_chain;
 extern crate reqwest;
-
-use std::io::Read;
-#
-# error_chain! {
-#     foreign_links {
-#         Io(std::io::Error);
-#         HttpRequest(reqwest::Error);
-#     }
-# }
-
-fn main() -> Result<()> {
-    let mut res = reqwest::get("http://httpbin.org/get")?;
-    let mut body = String::new();
-    res.read_to_string(&mut body)?;
-
+#[tokio::main]
+async fn main() -> Result<(), reqwest::Error> {

```

Comment:
error[E0670]: `async fn` is not permitted in the 2015 edition
 --> /tmp/rust-skeptic.rEEf54DF9Cer/test.rs:4:1
  |
4 | async fn main() -> Result<(), reqwest::Error> {
  | ^^^^^
error[E0609]: no field `await` on type `impl std::future::Future`
 --> /tmp/rust-skeptic.rEEf54DF9Cer/test.rs:5:54
  |
5 |     let res = reqwest::get("http://httpbin.org/get").await?;
  |                                                      ^^^^^
  |
  = note: to `.await` a `Future`, switch to Rust 2018
  = help: set `edition = "2018"` in `Cargo.toml`
  = note: for more on editions, read https://doc.rust-lang.org/edition-guide
error: aborting due to 2 previous errors
Some errors have detailed explanations: E0609, E0670.
For more information about an error, try `rustc --explain E0609`.
test get_sect_make_a_http_get_request_line_10 ... FAILED

Reply:
Looks like this work is blocked by #530 which ideally is a change in error-chain to push through the PR.  You should be able to add edition2018 to the infostring:
\```rust,edition2018

Reply:
rust,edition2018 doesn't work to me :S , setting it at the `Cargo.toml` as error report didn't break any other test.